processing/src/main/java/io/druid/granularity/PeriodGranularity.java

```diff
@@ -45,6 +46,7 @@ public PeriodGranularity(
   )
   {
     this.period = period;
```

Comment:
might as well toss `this.period = Preconditions.checkNotNull(period, "period");` while we're here.